### ISSUE TITLE
Confirm the votes are correct for the TEC MVV Vote

### ISSUE THREAD

**Issue by GriffGreen** [wslyvh/tokenlog#31]
<img width="503" alt="Screen Shot 2021-04-25 at 9 16 02 AM" src="https://user-images.githubusercontent.com/14945613/116000911-616af400-a5f2-11eb-90b0-75b4caaa1817.png">

Michael Zargham, [25.04.21 09:10]
[ Album ]
something is buggy with the token log -- 

first off #16 MVV had ~450 votes or so before i voted. secondly, i only intended to cast 100 votes (at a cost of 10,000), due to the ux of the tab not closing, i actually did it twice, not the end of the world but it should have blocked that because i don't have 200^2 = 40,000 to spend. beyond that the total count jumped to 798 which seems really weird to me.

Griff Green🌱, [25.04.21 09:14]
Maybe other people voted at the same time?

Michael Zargham, [25.04.21 09:14]
that would account for the jump to 798, but not the fact that i inadvertantly sybiled by applying 100 votes twice

Griff Green🌱, [25.04.21 09:15]
K I will report to @wslyvh to confirm that the math is right

Griff Green🌱, [25.04.21 09:15]
and to see if we can make sure it works

Michael Zargham, [25.04.21 09:16]
as far as i can tell could actually do it again -- add 100 more votes at a cost of 10,000 since i still have voting capacit to spend

**Comment by GriffGreen** [comment]
I think I recreated the issue... 

I am assuming the votes are correct... but basically after voting once, the vote button still works and initiates the signature for using the same numbers... but i don't think it increments the vote count

Below is a screen shot where I voted already, notice i only have 317 vp left... so that updated... but i was still able to push the vote button again and sign another message... giving me the experience of frauding the system... even tho it doesnt seem to increment the vote total

<img width="1036" alt="Screen Shot 2021-04-26 at 8 48 06 PM" src="https://user-images.githubusercontent.com/14945613/116184600-5e6e2180-a720-11eb-8ae2-f4dd08378847.png">

**Comment by wslyvh** [comment]
Thanks for the screenshots and detailed steps to reproduce.
I noticed casting the vote itself takes quite some time, so people tended to just vote again (using same votes/cost).
It was indeed possible to vote multiple times this way and mess up the results.

Few improvements that have been added:
- disabled voting after casting your initial vote
- added status indicators and friendly error messages. See #32 
- added statistics and all raw voting data, so projects can verify the results 
e.g. https://tokenlog.xyz/CommonsBuild/TokenLog-SoftGov/stats

Including the additional checks I added previously #24, I think this shouldn't be possible anymore.
Closing this issue, but do reach out if you still experience similar behavior
